nebula_decoders/include/nebula_decoders/nebula_decoders_hesai/decoders/hesai_decoder.hpp

```diff
@@ -247,7 +247,7 @@ class HesaiDecoder : public HesaiScanDecoder
         sensor_configuration_->scan_phase * SensorT::packet_t::DEGREE_SUBDIVISIONS);
 
       if (scan_completed) {
-        std::swap(decode_pc_, output_pc_);
+          std::swap(decode_pc_, output_pc_);
```

Comment:
Tabbing oops?

Reply:
Addressed in [28b42fc](https://github.com/tier4/nebula/pull/127/commits/28b42fcb66ceedae689521f3fc5212f3bd3b35f0)